src/peft/tuners/adalora/bnb.py

```diff
@@ -85,37 +74,28 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
 if is_bnb_4bit_available():
 
-    class SVDLinear4bit(bnb.nn.Linear4bit, AdaLoraLayer):
+    class SVDLinear4bit(torch.nn.Module, AdaLoraLayer):
```

Comment:
This might be breaking no? Some setups might consider some checks such as `isinstance(module, bnb.nn.Linear4bit)`, suddenly these layers will not inherit anymore from bnb layers

Reply:
ok sounds great yeah, I think this is too much of an edge case, let's just be careful on the release notes and maybe add this PR on the next minor release (0.7.0)